RAT_README.txt

```diff
@@ -60,6 +60,7 @@ $MY_SQROOT/../sql/bin/SqlciErrors.txt
 
 dcs/conf/servers 
 dcs/conf/backup-masters 
+dcs/conf/master
                  :  Apache Trafodion DCS default configuration file
                  -> configuration file do not handle comments
 
```

Comment:
could you please list out what those three files for separately. "Apache Trafodion DCS default configuration file" is not clear.